apps/site/assets/ts/helpers/use-params.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Basically replacement for use-query-params, query-param, similar packages
+ */
+
+export const getParam = (key: string): string | null | undefined => {
+  const params = new URLSearchParams(window?.location.search ?? undefined);
+  return params.get(key);
+};
+
+export const updateParams = (
+  updatedParams: Record<string, string | null>
+): void => {
+  const params = new URLSearchParams(window?.location.search ?? undefined);
+  Object.entries(updatedParams).forEach(([key, value]) => {
+    if (value) {
+      params.set(key, value);
+    } else {
+      params.delete(key);
+    }
+  });
+
+  window?.history.replaceState(
+    {},
+    "",
+    `${window.location.pathname}?${params.toString()}`
+  );
+};
+
+export default {
```

Comment:
```suggestion
export {
```
suggestion: I don't think you want `default` for named exports.